### DISCUSSION TITLE
Maximum size for SD card? and will a MicroSD in a SD adapter work?

### DISCUSSION THREAD

**Discussion by dburr** [BlueSCSI/BlueSCSI-v2#67] [Help] [answered]
Trying to get to the bottom of some `scsi_accel_rp2040_finishWrite timeout` errors I am getting, and while Google is not coming up with anything if I use that specific search term, I do see in the Troubleshooting guide that `scsi_accel_rp2040_finishRead timeout` could be caused by a too-slow SD card. My SD cards are indeed rather old, so I thought it might be a good idea to replace them. However all I can seem to find nowadays are MicroSD cards. Will a (good quality name brand, not an off-brand) MicroSD card in a SD adapter work? And is there a limit to the size of card that will work? Pretty much the smallest name-brand card I can get nowadays is 32 GB, and most are in the 64-128 GB range. Is there a problem in using these larger cards on a BlueSCSI v2? (other than wasting card space, because none of my HD images comes anywhere close to filling up a card of that size)

**Comment by erichelgeson** [comment, marked as answer]
There is no problems using large cards. I use 512gb. People have used 1tb.
